<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
<meta http-equiv="Content-Style-Type" content="text/css" />
<meta http-equiv="Content-Script-Type" content="text/javascript" />
<meta http-equiv="Content-Language" content="en-us" />
<meta name="Description" content="Protein hydration" />
<meta name="Keywords" content="protein hydration,protein" />
<meta name="author" content="martin chaplin: martin.chaplin@btinternet.com" />

<title>Protein hydration</title>
<script language="javascript" type="text/javascript" src="head.js"></script>
</head>
<noscript>
Your browser does not support JavaScript!
</noscript>
<head>
<link rel="shortcut icon" href="/water/favicon.ico" type="image/x-icon" />
<link rel="ToC" href="/water/water_structure_science.html" />
<link rel="stylesheet" href="water.css" type="text/css" />

</head>
<body onload="mm_preloadImages('images/seab1.gif','images/seab3.gif','images/seab4.gif','images/top2.gif', 'images/top3.gif')"> 
<a name="top" id="top"></a>
<div style="width:761px; height:93px; background-color:#006868;padding-top:3px;border:0px;margin:0px;">
  <div style="padding:0px;border:0px;margin:0px;padding-left:3px;width:140px; height:22px; background-color:#006868; float:left; display:inline;">
    <form class="quicklinkform" action=" ">
      <label for="&rdquo;menu1&rdquo;"></label>
      <select class="quicklink"
            onchange="mm_jumpMenu('parent',this,0)" name="menu1" id="&rdquo;menu1&rdquo;">
       <option  value="index.html" selected="selected">Quick links</option>
        <option value="index.html">......................................</option>
        <option value="water_molecule.html">Water molecule</option>
        <option value="water_hydrogen_bonding.html">hydrogen-bonding</option>
        <option value="water_phase_diagram.html">Phase diagram</option>
        <option value="water_vibrational_spectrum.html">Water spectrum</option>
        <option value="water_dissociation.html">Dissociation</option>
        <option value="water_anomalies.html">Anomalies</option>
        <option value="water_properties.html">Water properties</option>
        <option value="index.html">......................................</option>
        <option value="clusters_overview.html">Water clusters</option>
        <option value="protein_hydration.html">Protein hydration</option>
        <option value="hydrocolloids_gums.html">Hydrocolloids</option>
        <option value="ion_hydration.html">Ions</option>
        <option value="index.html">......................................</option>
        <option value="water_structure_science.html">Table of Contents</option>
        <option value="water_sitemap.html">Site map</option>
</select>
</form>
</div>
  <div style="padding:0px;border:0px;margin:0px; height:22px; background-color:#006868; float:left; display:inline;"><a href="search.html" onmouseout="mm_swapImgRestore()" onmouseover="mm_swapImage('search','','images/seab1.gif',1)"><img src="images/sear1.gif" border="0" name="search" id="search"  width="72" height="22" title="Go to Search page" alt="Water site header" /></a><a href="water_sitemap.html"><img src="images/sear2.gif" width="432" height="22" title="Masthead Island, Great Barrier Reef: Go to Water Site map" alt="Masthead Island, Great Barrier Reef" border="0" /></a><img src="images/sean3.gif" alt=" " width="36" height="22" /><a href="/php-cgiwrap/water/pfp.php3?page=http://water.lsbu.ac.uk/water/protein_hydration.html" onmouseout="mm_swapImgRestore()" onmouseover="mm_swapImage('printer','','images/seab4.gif',1)"  ><img src="images/sear4.gif" width="36" height="22"  title="Click here to print page" alt="Print-me key" name="printer"  border="0" id="printer" /></a><a href="add1.html"><img src="images/sear5.gif" width="39" height="22"  title="go to Water Visitor Book contributions"  alt="go to Water Visitor Book contributions" border="0"  /></a></div>
  <map name="topmap" id="topmap">
    <area shape="poly" coords="322,42,429,42,429,61,322,61" href="martin_chaplin.html" title="Go to my page" alt="Go to my page" onmouseout="mm_swapImgRestore()" onmouseover="mm_swapImage('topi','','images/top3.gif',1)"  />
      <area shape="poly" coords="553,3,554,30,693,30,717,8,747,32,717,56,690,33,204,35,204,3" href="water_structure_science.html" title="Go to Water Structure and Science site contents" alt="Water Structure and Science" onmouseout="mm_swapImgRestore()" onmouseover="mm_swapImage('topi','','images/top2.gif',1)" />
</map>
  <div style="padding:0px;border:0px; margin-left:3px; width:755px; background-color:#006868;"><img src="images/top1.gif" width="755" height="68" name="topi" id="topi" usemap="#topmap" border="0" alt=" " /></div>
</div>
<div style="width:761px; background-color:#006868;padding:0px;border:0px;margin:0px;">
  <div style="width:751px; background-image:url('images/bg.gif');margin-left:3px;padding-left:4px;border:0px;padding-top:0px;padding-bottom:0px; background-color:#FFFae2;">
    <!-- print begin -->    
<figure  class="floatright">
<figcaption>
      <p align="center">&nbsp;</p>
      <p align="center">Human lysozyme, 1GF8. </p>
      <p align="center" class="style4">The water molecules are not correctly directed</p>
      <p align="center" class="style4">&nbsp;</p>
</figcaption>
<img src="images/human_lysozyme.gif" title="Human lysozyme, 1GF8. The water molecules are not correctly directed." alt="Human lysozyme, 1GF8. The water molecules are not correctly directed" width="320" height="253" class="floatright" /></figure>
    <h1>Protein 
      Hydration</h1>
    <p>Water is a significant contributor to a protein's 3-D structure, and the protein controls the structuring of its surrounding water.</p>
    <p>&nbsp;</p>
    <p><span class="style16"><img src="images/redleft.gif" alt="V" width="12" height="13" border="0"/> <a href="protein_denatured.html" title="go for more information">Protein folding and denaturation</a></span></p>
    <p><span class="style16"><img src="images/redown.gif" alt="V" width="13" height="12" border="0"/> <a href="#cont" title="go to section below">The contribution of water to protein 
      structure</a></span><span class="style16"><br />
      <img src="images/redown.gif" alt="V" width="13" height="12" border="0"/> <a href="#rec" title="go to section below">Water in protein recognition and binding </a><br />
      <img src="images/redown.gif" alt="V" width="13" height="12" border="0"/> <a href="#func" title="go to section below">Water in protein and enzyme function </a></span></p>
    <p><a href="#bip"><span class="style16"><img src="images/redown.gif" alt="V" width="13" height="12" border="0"/> Enzymic reactions in biphasic liquid systems</span></a></p>
    <p><span class="style16"><a href="#AFP"><img src="images/redown.gif" alt="V" width="13" height="12" border="0"/> Antifreeze proteins </a><br />
  </span></p>
    <p>&nbsp;</p>
     <div class="style10">
    <p align="center"><em>'Interactions with water govern the folding, structure, stability and  activity of proteins'</em></p>
</div>
    <h2><a name="cont" id="cont"></a>The contribution of water to protein 
      structure</h2>
    <p>Protein hydration is essential for their three-dimensional structure, the dynamic ensemble of conformations  [<a title="go to cited reference" href="ref23.html#r2249">2249</a><a name="r2249" id="r2249"></a>, <a title="go to cited reference"  href="ref30.html#r2921">2921</a><a name="r2921" id="r2921"></a>], and activity [<a title="go to cited reference" href="ref5.html#r472">472</a><a name="r472" id="r472"></a>, <a title="go to cited reference" href="ref11.html#r1093">1093</a><a name="r1093" id="r1093"></a>, <a title="go to cited reference" href="ref14.html#r1345">1345</a><a name="r1345" id="r"></a>, <a title="go to cited reference" href="ref21.html#r2005">2005</a><a name="r2005" id="r2005"></a>]. A perspective of the use of X-ray and neutron scattering and diffraction, NMR, terahertz spectroscopy, and molecular simulations has been presented [<a title="go to cited reference" href="ref39.html#r3830">3830</a>]<a name="r3830" id="r3830"></a>. A structural determination with a resolution of about 1.5&ndash;1.6 <a href="constants.html#ang" title="Angstrom">&Aring;</a> is necessary to observe the continuous hydration layer at the protein surface  [<a title="go to cited reference" href="ref31.html#r3049">3049</a><a name="r3049" id="r3049"></a>], which extends to thousand(s) of water molecules. Fluctuations of the protein surface groups drive and are driven and controlled by the surrounding network of water molecules [<a title="go to cited reference" href="ref27.html#r2648">2648</a><a name="r2648" id="r2648"></a>, <a title="go to cited reference" href="ref30.html#r2917">2917</a><a name="r2917" id="r2917"></a>, <a title="go to cited reference"  href="ref43.html#r4202">4202</a><a name="r4202" id="r4202"></a>]. Indeed, proteins lack biological activity without sufficient hydrating water (usually at least a mono-layer covering; &gt; 1.5 mols H<sub>2</sub>O mol<sup>&minus;1</sup> amino acid residue). High-resolution X-ray diffraction detects about one H<sub>2</sub>O mol<sup>&minus;1</sup> amino acid that isrelatively static [<a title="go to cited reference" href="ref27.html#r2677">2677</a><a name="r2677" id="r2677"></a>]. The aqueous structuring around proteins is affected out to at least 1 - 1.5 nm from its surface or 2 - 3 nm between neighboring proteins, as shown by <a  title="go for more information" href="water_methods.html#tera">terahertz spectroscopy</a> [<a title="go to cited reference" href="ref14.html#r1368">1368</a><a name="r1368" id="r1368"></a>, <a title="go to cited reference" href="ref22.html#r2102">2102</a><a name="r2102" id="r2102"></a>] and molecular dynamics [<a title="go to cited reference" href="ref29.html#r2886">2886</a><a name="r2886" id="r2886"></a>],<sup><a href="#e" name="be" id="be"> e</a></sup> with even small proteins (e.g., bovine serum albumin,  66,463&nbsp;Da) affecting the whole of its unstirred <a href="colligative_generation.html#g">(Nernst) layer</a> of about 20,000 neighboring water molecules [<a title="go to cited reference" href="ref22.html#r2102">2102</a>]. The free energy of hydrogen-bond breakage around proteins has been found to be about 1.7 kJ  &#739; mol<sup>&minus;1</sup> higher than that of bulk water [<a title="go to cited reference"  href="ref43.html#r4206">4206</a><a name="r4206" id="r4206"></a>]. </p>
    <p>&nbsp;</p>
    <p>Indeed, the range of these interactions is governed by directed hydrogen-bonding interactions that extend further than the electrostatics-dependent Debye screening length (&lt; 1 nm). Additionally, the presence of glycans attached to (glyco)proteins imposes a long-range order on the water structure out to several nanometers, dependent on the orientation of the glycan [<a title="go to cited reference" href="ref22.html#r2104">2104</a><a name="r2104" id="r2104"></a>]. </p>
    <p>&nbsp;</p>
    <p>Some water molecules  interact with the surface, reorienting both themselves and the surface groups, whereas other water molecules link these to the bulk in an ordered manner while remaining in dynamically active [<a title="go to cited reference" href="ref17.html#r1695">1695</a><a name="r1695" id="r2"></a>]. The closer the water molecules are to the surface, the more likely that movement will be due to hopping between hydrogen bonding sites rather than conventional diffusion. The interfacial dielectric constant of these interfacial water molecules is about 2&ndash;4  reflecting the constraints imposed by the protein on the hydration waters and their low ability to polarize in response to an external field [<a title="go to cited reference"  href="ref35.html#r3493">3493</a><a name="r3493" id="r3493"></a>]. The surface of a protein has more hydrogen-bond acceptor groups (e.g., -O-, =O, =N-) than  hydrogen-bond donor groups ( e.g., O-H, =N-H, -NH<sub>2</sub>, -NH<sub>3</sub><sup>+</sup>). This, together with excluded-volume effects, necessarily causes several vacant hydrogen-bonding sites around the local water molecules. In solution, 
      proteins possess  conformational flexibility,  encompassing 
      a wide range of hydration states, not seen in the crystal<sup><a href="#a" name="ba" id="ba"> a</a></sup> or in non-aqueous environments. The equilibrium between these 
      states will depend on the <a href="water_activity.html">activity</a> of the water within its micro-environment.that is, the 
    freedom that the water has to hydrate the protein [<a title="go to cited reference" href="ref5.html#r434">434</a><a name="r434" id="r434"></a>]. </p>
    <p>&nbsp;</p>
    <p>Protein conformations demanding greater hydration are 
      favored by more (re-)active water (for example, high-density 
      water containing many weak, bent, or broken hydrogen bonds) 
      and 'drier' conformations are relatively favored by lower 
      activity water (for example, low-density water containing 
      many strong intra-molecular aqueous hydrogen bonds). Surface 
      water molecules are held to each other most strongly by the positively charged 
      basic amino acids. The exchange of surface water (and hence 
      the perseverance of the local clustering and the overall system 
      flexibility) is controlled by the exposure of the groups to 
      the bulk solvent (that is, greater exposure correlates 
      to greater flexibility and more  unrestrained  protein chain movement) [<a title="go to cited reference" href="ref10.html#r975">975</a><a name="r975" id="r975"></a>]. The structuring and dynamics of the  inner hydration layers are affected by the protein surface chemical composition (hydrophobic, hydrogen-bonding, polar, acidic, and basic groups), the complex topography (clefts, grooves, pockets, and protrusions), the presence of water 'wires' and the requirements of the secondary hydration layers' hydrogen-bonding [<a title="go to cited reference" href="ref28.html#r2781">2781</a><a name="r2781" id="r2781"></a>]. Counter-intuitively, hydrating water is slower near hydrophobic patches than near hydrophilic residues. This is due to stronger confinement often found in these areas. Protein conformational fluctuations affect the hydration shell dynamics in a heterogeneous manner across the protein surface. They particularly modify and reduce the  aqueous dynamics within concave regions of partial confinement [<a title="go to cited reference" href="ref28.html#r2725">2725</a><a name="r2725" id="r2725"></a>]. Although usually destabilizing, the presence of a surface bulky hydrophobic residue may sometimes increase the thermal stability of a protein [<a title="go to cited reference"  href="ref37.html#r3694">3694</a><a name="r3694" id="r4"></a>]. The structures of six bovine pancreatic trypsin inhibitors (BPTI)-[5,55]Gly<sup>14</sup> variants (Gly<sup>14</sup>Gly<sup>38</sup>, Gly<sup>14</sup>Ala<sup>38</sup>, Gly<sup>14</sup>Val<sup>38</sup>, Gly<sup>14</sup>Leu<sup>38</sup>, Gly<sup>14</sup>Ile<sup>38</sup>, and Gly<sup>14</sup>Lys<sup>38</sup>) all retained essentially the same structure as the wild-type BPTI. However, they showed an increase in thermal stability coinciding with their increase in surface hydrophobicity. The number of water molecules near residue 38 increased upon its mutation to a hydrophobic residue, suggesting that improved hydration contributed to the enthalpy-driven stabilization. [<a title="go to cited reference"  href="ref37.html#r3694">3694</a>]    </p>
    <p>&nbsp;</p>
    <p>Hydration also affects the reactions and interactions of co-enzymes and cofactors; thus, the various redox potentials (and hence whether they oxidize or reduce) of some iron-sulfur proteins  are accounted for by differential hydration rather than direct protein binding effects [<a title="go to cited reference" href="ref14.html#r1344">1344</a><a name="r1344" id="r1344"></a>]. Hydration of ionic solutes shields them from the protein surface, such that  <a href="water_dissociation.html#o">ionic strength</a> changes have little effect on protein charge distribution  [<a title="go to cited reference" href="ref23.html#r2278">2278</a><a name="r2278" id="r2278"></a>]. </p>
    <p>&nbsp;</p>
    <p>The folding of proteins depends on the same factors as control 
      the junction zone formation in some <a href="polysaccharide_hydration.html">polysaccharides</a>; that is, the incompatibility between the low-density 
      water (<abbr title="low-density water">LDW</abbr>) and 
      the hydrophobic surface that drives such groups to form the 
      hydrophobic core.<sup><a href="#b" name="bb" id="bb"> b</a></sup> This drive for hydrophobic groups to mostly cluster away from the protein surface (in water-soluble proteins) is controlled by the charged and polar group interactions with each other and  their cooperative hydration  [<a title="go to cited reference" href="ref28.html#r2716">2716</a><a name="r2716" id="r2716"></a>]. It is challenging to model this driving force   quantitatively [<a title="go to cited reference" href="ref25.html#r2491">2491</a><a name="r2491" id="r2491"></a>, <a title="go to cited reference" href="ref25.html#r2492">2492</a><a name="r2492" id="r2492"></a>]. Dehydration of Concanavalin A induces the decrease of its &beta;-sheet secondary structure due to the breakage of their &beta;-sheet-linking hydrogen -bonds  [<a title="go to cited reference" href="ref29.html#r2889">2889</a><a name="r2889" id="r2889"></a>]. </p>
    <p>&nbsp;</p>
    <p>Interestingly,  some buffer-insoluble proteins are quite soluble in pure water and  the  absence of screening dissolved ions due to the weak (here, unshielded)  interactions between the protein's intrinsic charges  [<a title="go to cited reference" href="ref14.html#r1375">1375</a><a name="r1375" id="r1375"></a>]. Ions have effects on both protein solubility and protein stability (see the <a href="hofmeister_series.html">Hofmeister Series</a>). Non-ionic <abbr title="Collapsed icosahedral water cluster structure"><a href="kosmotropes_chaotropes.html" title="more on kosmotropes and chaotropes">kosmotropes</a></abbr>, 
      which stabilize low-density water, consequentially stabilize the structure of proteins. In addition, 
      water acts as a lubricant [<a title="go to cited reference" href="ref9.html#r822">822</a><a name="r822" id="r822"></a>], 
      so easing the necessary peptide amide-carbonyl hydrogen-bonding 
      changes. The biological activity of proteins appears to depend 
      on forming a 2-D hydrogen-bonded network spanning 
      most of the protein surface and connecting all the surface 
      hydrogen-bonded water clusters [<a title="go to cited reference" href="ref10.html#r978">978</a><a name="r978" id="r978"></a>]. 
      Such a water network can transmit information around 
      the protein and control its dynamics, such as its 
      domain motions [<a title="go to cited reference" href="ref10.html#r977">977</a><a name="r977" id="r977"></a>]. In particular, the translational diffusion of the interfacial water molecules  stimulates the large-amplitude protein motions required for  biological activity [<a title="go to cited reference" href="ref25.html#r2435">2435</a><a name="r2435" id="r2435"></a>]. Hydrating water molecules can control the functional motions of multi-domain proteins. For example, each subunit in  hexameric glutamate dehydrogenase (GDH)  comprises a nucleotide-binding domain (N-domain) and a core domain (C-domain) for hexamer formation [<a title="go to cited reference" href="ref26.html#r2583">2583</a><a name="r2583" id="r2583"></a>]. This molecular dynamics (MD) simulation shows that cooperative changes   in the hydration structure within the cleft between the domains switches domain-closure on or off, with the hydrophobic closure   necessary for substrate binding. These findings are reinforced by experimental observations  that accompany &mu;s-ms motions in dihydrofolate reductase [<a title="go to cited reference" href="ref32.html#r3166">3166</a><a name="r3166" id="r3166"></a>].  In this protein, changes in hydration appear to be necessary for collective domain motions, with water molecules  gating structural transitions by wetting and drying protein cavities, leading to changes in the local hydrogen-bonding networks. This study also indicated that many of the indoles in proteins are hydrated even though the individual lifetimes of waters may be very short. These water molecules  play a critical role in transmitting information between functionally essential  regions of the protein and provide evidence that internal protein motions can be coupled through the mediation of hydrogen-bonded water bound in the protein structure. A functional switch between a wet and a dry configuration also occurs in <em>Rhodobacter sphaeroides</em> cytochrome c oxidase [<a title="go to cited reference" href="ref32.html#r3186">3186</a><a name="r3186" id="r3186"></a>]. This transition occurs in the proton&rsquo;s exit channel. The protonation state of its acidic Glu<sub>286</sub> residue is controlled by switching   between the dry and wet states. In the wet state, this p<em>K</em><sub>a</sub>  is lowered to facilitate this protons transfer to the cytochrome a<sub>3</sub>-Cu<sub>&Beta;</sub> binuclear center.</p>
    <p>&nbsp;</p>
    <p>Intramolecular peptide (amide) hydrogen bonding makes a significant 
      contribution to protein structure and stability. Equilibrium H/D fractionation may be used to assess hydrogen bond strengths quantitatively and has shown them to be marginally stronger on average in &alpha;-helices rather than &beta;-sheets by &asymp; 0.8 kJ  &#739; mol<sup>&minus;1</sup> [<a title="go to cited reference" href="ref22.html#r2115">2115</a><a name="r2115" id="r2115"></a>]. However, such hydrogen bonds are only 
      effective in the absence of accessible competing water. Even 
      just the presence of close-by water molecules causes peptide hydrogen bonds to lengthen [<a title="go to cited reference" href="ref6.html#r524">524</a><a name="r524" id="r524"></a>], 
      so loosening the structure. Water molecules can bridge between 
      the carbonyl oxygen atoms and amide protons of different peptide 
      links to catalyze the formation, and its reversal, of peptide 
      hydrogen-bonding as well as forming long-lived linkages stabilizing 
      protein-ligand and protein-protein interfaces (for example, [<a title="go to cited reference" href="ref7.html#r688">688</a><a name="r688" id="r688"></a>]). The 
      internal molecular motions in proteins, necessary for biological 
      activity, are very dependent on the degree of plasticizing, 
      which is determined by the level of hydration [<a title="go to cited reference" href="ref9.html#r810">810</a><a name="r810" id="r810"></a> (greater water gives lower <a href="amorphous_ice.html#e" title="go for an explanation" >glass</a> transition temperature and greater plasticity [<a title="go to cited reference"  href="ref23.html#r2245">2245</a><a name="r2245" id="r2245"></a>]). 
      Thus internal water enables the folding of proteins and is 
      only expelled from the hydrophobic central core when finally 
      squeezed out by cooperative protein chain interactions [<a href="ref4.html#r352" name="r352" title="go to cited reference" id="r352">352</a>]. 
      A method for computing buried waters  and multi-water bridges  in molecular dynamics trajectories has been described [<a title="go to cited reference"  href="ref38.html#r3701">3701</a><a name="r3701" id="r3701"></a>]. Many water molecules (similar in amount to individual amino 
      acids) remain  buried in the core of the proteins,  
      forming structurally-important hydrogen-bonded linkages. The 
      position of the <abbr title="Expanded icosahedral water cluster structure"><a title="go to 'Water clusters' page"  href="icosahedral_water_clusters.html#ES">ES</a></abbr><img border="0" src="images/rarrow.gif" width="20" height="12"  alt="reversible arrow" /><abbr title="Collapsed icosahedral water cluster structure"><a title="go to 'Water clusters' page"  href="icosahedral_water_clusters.html#CS">CS</a></abbr> equilibrium around enzymes has been shown to be essential 
      for their activity with the enzyme balanced between flexibility 
      (<abbr title="Collapsed icosahedral water cluster structure"><a title="go to 'Water clusters' page"  href="icosahedral_water_clusters.html#CS">CS</a></abbr> environment) and rigidity 
    (<abbr title="Expanded icosahedral water cluster structure"><a title="go to 'Water clusters' page"  href="icosahedral_water_clusters.html#ES">ES</a></abbr> environment). Incoherent neutron scattering has been used to assess the protein's structural rigidity [<a title="go to cited reference"  href="ref40.html#r3920">3920</a><a name="r3920" id="r3920"></a>].</p>
    <p>&nbsp;</p>
    <p>The addition of non-ionic <a href="kosmotropes_chaotropes.html" title="more on kosmotropes and chaotropes">chaotropes</a> only, or <a href="kosmotropes_chaotropes.html" title="more on kosmotropes and chaotropes">kosmotropes</a> only,  
      may inhibit the activity of enzymes (by shifting the equilibrium 
      to the right or left, respectively), whereas an intermediate 
      mixture of <a href="kosmotropes_chaotropes.html" title="more on kosmotropes and chaotropes">kosmotropes</a> and <a href="kosmotropes_chaotropes.html" title="more on kosmotropes and chaotropes">chaotropes</a> restores optimum activity [<a href="ref3.html#r276" name="r276" title="go to cited reference" id="r276">276</a>]. 
      Also, the ease of enzyme-substrate contact may be controlled 
      by the level of water structuring in the protein environs 
      [<a title="go to cited reference" href="ref9.html#r882">882</a><a name="r882" id="r882"></a>]. Water in the crystal structure can be positioned by diffraction at high resolution (0.15 - 0.16 nm), with less being seen at lower resolutions [<a title="go to cited reference" href="ref26.html#r2554">2554</a><a name="r2554" id="r2554"></a>]. The crystal structures of proteins suggest that the commonest water polygons surrounding proteins are (H<sub>2</sub>O)<sub>3</sub> trimers (typical of <abbr title="Collapsed icosahedral water cluster structure"><a title="go to 'Water clusters' page"  href="icosahedral_water_clusters.html#CS">CS</a></abbr> environments) and (H<sub>2</sub>O)<sub>5</sub> pentamers (typical of <abbr title="Expanded icosahedral water cluster structure"><a title="go to 'Water clusters' page"  href="icosahedral_water_clusters.html#ES">ES</a></abbr><abbr title="Collapsed icosahedral water cluster structure"></abbr> environments) with lower amounts of tetramers, hexamers, and heptamers  
    [<a title="go to cited reference" href="ref19.html#r1849">1849</a><a name="r1849" id="r1849"></a>].</p>
    <p>&nbsp;</p>
    <p> Much of life functions optimally at about 37&nbsp;&deg;C, where a spanning hydration network is about to break down on heating, perhaps to help compensate for the biosystem's entropic changes [<a title="go to cited reference" href="ref16.html#r1541">1541</a><a name="r1541" id="r1541"></a>].</p>
    <p>&nbsp;</p>
    <p>Water-mediated hydrogen bonding between peptide links has been found to be particularly important in the structure of collagen where only a third of the peptide links directly hydrogen bond  to other peptide links [<a title="go to cited reference" href="ref14.html#r1314">1314</a><a name="r1314" id="r1314"></a>], and the stability of the triple helix is sequence-dependent  [<a title="go to cited reference" href="ref15.html#r1438">1438</a><a name="r1438" id="r1438"></a>]. The single-water bridges found within the individual triple-helix collagen molecules have been found to be dynamically linked with freely exchangeable hydrogen atoms rather than being ice-like     [<a title="go to cited reference" href="ref29.html#r2840">2840</a><a name="r2840" id="r2840"></a>]. The collagen matrix  regulates the delivery of calcium phosphate species for bone mineralization and provides the spatial volume for intrafibrillar mineral deposition. Water in the nanoconfined spaces within the fibrils is quite different from bulk water and has a much lower density (&asymp; 0.70 g   &#739; cm<sup>&minus;3</sup> ). It influences the  infiltration of calcium phosphate species into the collagen fibril and subsequent dehydration to form a solid amorphous phase      [<a title="go to cited reference" href="ref32.html#r3148">3148</a><a name="r3148" id="r3148"></a>]. </p>
    <p>&nbsp;</p>
    <p>The first hydration shell around proteins (&asymp; 0.3 g/g) is ordered; 
      with high proton transfer rates and well-resolved time-averaged 
      hydration sites; surface water showing coherent (orderly) <a title="go to 'Hydrogen bonding in water' page"  href="water_hydrogen_bonding.html">hydrogen-bond</a> patterns with large net dipole fields [<a title="go to cited reference" href="ref8.html#r702">702</a><a name="r702" id="r702"></a>]. 
      Such hydrating sites may be mapped by calculation [<a title="go to cited reference" href="ref41.html#r4033">4033</a><a name="r4033" id="r4033"></a>] to assist structure-based drug design. The hydrogen bonds holding these water molecules to the protein are stronger with longer lifetimes than bulk water [<a title="go to cited reference" href="ref14.html#r1355">1355</a><a name="r1355" id="r1355"></a>], and this water is unavailable to <a title="go to 'Colligative properties' page"  href="colligative_properties.html">colligative effects</a>. By  retaining some liquidity, this hydration shell protects the protein against  irreversible denaturing effects, such as very low temperatures  [<a href="ref24.html#r2338" name="r2338" title="go to cited reference" id="r2338">2338</a>]. As hydration sites may be positioned close together and therefore 
      mutually exclusive, it has been argued that the solvation 
      is better described as a water distribution density function 
      rather than by specific water-occupied sites. For example, 
      no more than 164 of the 294 high-density hydration sites around 
      myoglobin are occupied at any moment, and there is no correlation 
      between the maximum site density, occupancy, and residence 
      time [<a title="go to cited reference" href="ref6.html#r542">542</a><a name="r542" id="r542"></a>]. 
      The first hydration shell is also 10-20% denser than the bulk 
      water and probably responsible for keeping the molecules sufficiently 
      separated  so they remain in solution [<a href="ref4.html#r315" name="r315" title="go to cited reference" id="r315">315</a>] 
      (that is, solutions are kinetically stable but often 
      thermodynamically unstable).<sup><a href="#c" name="bc" id="bc"> c</a></sup> Although a significant amount of this density increase has 
      been shown to be due to simple statistical factors dependent 
      upon the way that the surface is defined in depressions [<a title="go to cited reference" href="ref5.html#r401">401</a><a name="r401" id="r401"></a>], 
      much is due to a protein's structure, and the higher density mainly in pockets within, or at the surface of the protein molecule  [<a title="go to cited reference" href="ref40.html#r3952">3952</a><a name="r3952" id="r3952"></a>]. The excess of polar 
      hydration sites (tending to increase surface density)<sup><a href="#d" name="bd" id="bd">d</a></sup> easily counteracts the remaining nonpolar surface groups 
      (tending to produce low-density surface water). Protein surfaces are not just hydrophilic but also possess hydrophobic regions that interact poorly with water and help provide the driving force for the proteins to interact with other proteins, ligands, and molecules [<a title="go to cited reference" href="ref36.html#r3556">3556</a><a name="r3556" id="r3556"></a>]. The surface water 
      is required for the protein to show its biological function 
      as, without it, the necessary fast conformational fluctuations 
    cannot occur. The interfacial water network exhibits significant cooperativity with the protein's global hydration shell that behaves collectively [<a title="go to cited reference" href="ref30.html#r2908">2908</a><a name="r2908" id="r2908"></a>]. Thus proteins have no activity (and enter a glassy state, at about 220 K) when the surrounding water becomes predominantly low-density [<a href="ref12.html#r1197" name="r1197" title="go to cited reference" id="r1197">1197</a>]. Also, dry proteins remain rigid (glassy) until sufficient water is added to hydrate the protein's charged groups,  occurring at a water weight fraction of 0.05 for lysozyme at 25&nbsp;&deg;C  [<a href="ref23.html#r2240" name="r2240" title="go to cited reference" id="r2240">2240</a>].    </p>
    <p>&nbsp;</p>
    <p>Using X-ray analysis, the hydration shell shows 
      a wide range of non-random hydrogen-bonding environments and 
      energies. Proteins are formed from a mixture of polar and 
      nonpolar groups. Water is most well-ordered around the polar 
      groups where residence times are longer, but where they will interfere with water's natural hydrogen-bonding, than around nonpolar 
    groups where aqueous clathrate structuring may form  [<a href="ref26.html#r2578" name="r2578" title="go to cited reference" id="r2578">2578</a>]. The water molecules surrounding  the side chains of glycine, alanine, &alpha;-aminoisobutyric acid, &alpha;-aminobutyric acid, valine, and related hydrocarbons have been shown to resemble the  gas clathrate hydrates, with the cage directly hydrogen-bonded to the protein backbone's polar groups [<a title="go to cited reference"  href="ref41.html#r4021">4021</a><a name="r4021" id="r4021"></a>]. The dynamics of the mutual interaction of water and protein as shown by dielectric relaxation have also been reviewed [<a href="ref37.html#r3617" name="r3617" title="go to cited reference" id="r3617">3617</a>].</p>
    <p>&nbsp;</p>
    <p>The ordering created in the water surrounding proteins extends the proteins' electrostatic surfaces well away from their physical (that is, amino acid) surfaces giving them far greater electrostatic visibility to visiting ligands [<a title="go to cited reference" href="ref12.html#r1156">1156</a><a name="r1156" id="r1156"></a>]. This non-specific electrostatic effect of the water effect is additional to any specifically directed hydrogen-bonding that may extend away into the bulk from the surface hydrogen-bonding to the amino acid side-chains helps to determine the three-dimensional folding of the proteins. This particularly involves the acidic (aspartic acid, D; glutamic acid, E), basic (lysine, K; arginine, R), and hydrophilic amino acids (asparagine, N; glutamine, Q; serine, S; threonine T) but also involves the aromatic amino acids that can form &pi;&middot;&middot;&middot;H-O hydrogen bonds (histidine, H; phenylalanine, F; tyrosine, Y; tryptophan, W)  [<a href="ref31.html#r3069">3069</a><a name="r3069" id="r3069"></a>]. </p>
    <div class="floatright" style="width:400px;">
      <figure  class="floatright">
        <figcaption>
          <p align="center">&nbsp;</p>
          <p align="center">Molar volumes of the amino acids</p>
          <p align="center">&nbsp;</p>
        </figcaption>
      <img src="images/aamolvol.gif" alt="Molar volumes of the amino acids" title="Molar volumes of the amino acids" width="400" height="245" /></figure
       >
      <p>&nbsp;</p>
      <p class="style3">&nbsp;</p>
      <p class="style3">The line is the best straight line from the amino acid data but is expected</p>
      <p class="style3"> &nbsp;to lie on the low-density side of aqueous structuring.</p>
      <p class="style3">&nbsp;</p>
      <p class="style3">&nbsp;</p>
    </div>
    <p>&nbsp;</p>
    <p>The graph opposite  shows the molar volumes of the amino acids [<a href="ref11.html#r1063">1063</a><a name="r1063" id="r1063"></a>] (using the one-letter code). Leucine (L) has  the largest molar volume relative to that expected from its relative molecular mass (molecular weight) and forms low-density clathrate water structures if exposed to the solution. It is followed by isoleucine (I), valine (V), phenylalanine (F), lysine (K), and arginine (R). Aspartic acid (D) has  the smallest molar volume relative to that expected from its molecular mass and forms higher density  water structuring. It is followed by asparagine (N), glutamic acid (E), serine (S), cysteine (C), glutamine (Q), histidine (H), threonine (T), and glycine (G). The hydration properties of alanine (A), proline (P), methionine (M), tyrosine (Y), and tryptophan (W) are slightly kosmotropic. The relative hydrophobicities have been assessed [<a href="ref29.html#r2820">2820</a><a name="r2820" id="r2820"></a>] as    </p>
    <p>&nbsp;</p>
    <p align="center">D &lt; E &lt; K &lt; R &lt; Q &lt; H &lt; N &lt; S &lt; Y &lt; W</p>
    <p align="center"> &lt; T&lt; M &lt; F &lt; C &lt; G &lt; A &lt; I &lt; V &lt; L &lt; P </p>
    <div style=" float:none; clear:both;">
      <p>&nbsp;</p>
      <p>The water is slow 
        to exchange, showing the dynamic behavior of bulk water 25&nbsp;&deg;C 
        colder [<a href="ref2.html#r147" name="r147" title="go to cited reference" id="r147">147</a>]. Low-density 
        water (such as <a title="go to 'Water clusters' page"  href="icosahedral_water_clusters.html#ES"><abbr title="Expanded icosahedral water cluster structure">ES</abbr></a>) 
        is promoted [<a href="ref2.html#r148" name="r148" title="go to cited reference" id="r148">148</a>, <abbr title="Collapsed icosahedral water cluster structure"><a title="go to cited reference" href="ref3.html#r276">276</a></abbr>] 
        surrounding this dense hydration and polyelectrolyte double 
        layer (as described in the '<a href="polysaccharide_hydration.html#zones">Polysaccharide 
          hydration</a>' section). Nonpolar groups promote clathrate 
        structures [<a href="ref2.html#r153" name="r153" title="go to cited reference" id="r153">153</a>] (such 
        as <a title="go to 'Water clusters' page"  href="icosahedral_water_clusters.html#ES"><abbr title="Expanded icosahedral water cluster structure">ES</abbr></a>) 
        surrounded by denser water. It is no surprise, therefore that 
        the degree of hydrophobic hydration is correlated with the 
        hydration of the polar groups further away. As clathrate-type structures break down at higher temperatures, hydrophobic hydration shows greater temperature dependence than hydrophilic hydration [<a href="ref28.html#r2761" name="r2761" title="go to cited reference" id="r2761">2761</a>]. Clathrate shells contain loosely 
        held water with greater rotational freedom than in the bulk 
        [<a href="ref2.html#r139" name="r139" title="go to cited reference" id="r139">139</a>]. <a name="blue" id="blue"></a>However, under 
        favorable conditions, clathrate hydrophobic hydration may exert 
        pressure on nonpolar C-H bonds, pushing them in, so contracting 
        their bond length, and increasing their vibrational frequency. 
        This blue-shifting (that is, the vibration frequency increases and intensity reduces) 'push-ball' hydration [<a href="ref2.html#r149" name="r149" title="go to cited reference" id="r149">149</a>] 
        should not necessarily be thought of as 'typical' hydrogen-bonding even 
        if the CH&middot;&middot;&middot;OH<sub>2</sub> distances 
        are suitably close (see [<a title="go to cited reference" href="ref13.html#r1293">1293</a><a name="r1293" id="r1293"></a>]). However, they can be considered as part of a continuum 
        of hydrogen-bonding behavior, where sometimes the OH<sub>2</sub> behaves as a much more weakly interacting base than usual 
        and the C-H behaves with reversed dipolar behavior compared 
        with the more usual O-H hydrogen-bonding partners [<a href="ref7.html#r625" name="r625" title="go to cited reference" id="r625">625</a>].</p>
    </div>
    <p>&nbsp;</p>
    <p>There are significant differences in the directional rates 
      of water diffusion perpendicular and parallel to the protein 
      surface that are maximal at about 6 <a href="constants.html#ang" title="Angstrom">&Aring;</a> but still determinable 
      at 15 <a href="constants.html#ang" title="Angstrom">&Aring;</a> from the surface [<a title="go to cited reference" href="ref6.html#r542">542</a>]. 
      Evolutionary processes have  used the 
      organization in this water surrounding proteins to create 
      preferred diffusive routes and orientation for metabolites 
      and favored conformational changes and interactions. Such 
      diffusive paths lead to binding sites with their  helpful 
      hydration. It has been suggested that pressure waves formed 
      from flickering water clusters (for example, as described <a title="go to 'Water clusters' page"  href="icosahedral_water_clusters.html">elsewhere</a>) may link protein molecular 
      vibrations, so carrying information through the intracellular 
      milieu [<a title="go to cited reference" href="ref6.html#r549">549</a><a name="r549" id="r549"></a>], <a title="go to cited reference" href="ref27.html#r2678">2678</a><a name="r2678" id="r2678"></a>] and powering product movement between enzymes in biochemical 
      pathways [<a title="go to cited reference" href="ref7.html#r665">665</a><a name="r665" id="r665"></a>].    </p>
    <p>&nbsp;</p>
    <div style=" float:none; clear:both;">
      <h2><a name="rec" id="rec"></a>Water in protein recognition and binding</h2>
    <figure  class="floatright">
<figcaption>
      <p align="center">Barnase-barnstar protein, from PDB 1BRS</p>
      <p align="center">&nbsp;</p>
</figcaption>
<img class="floatright" alt="barnase-barnstar protein, from PDB 1BRS" title="barnase-barnstar protein, from PDB 1BRS" src="images/barnstar.gif" width="280" height="196" /></figure>
      <p>Water molecules form an integral part of most protein-protein [<a title="go to cited reference" href="ref14.html#r1339">1339</a><a name="r1339" id="r1339"></a>], protein-DNA [<a title="go to cited reference" href="ref14.html#r1340">1340</a><a name="r1340" id="r1340"></a>] and protein-ligand [<a title="go to cited reference" href="ref14.html#r1341">1341</a><a name="r1341" id="r1341"></a>, <a title="go to cited reference" href="ref35.html#r3447">3447</a><a name="r3447" id="r3447"></a>] interactions, aiding the mutual recognition and both the binding thermodynamics and binding kinetics [<a title="go to cited reference" href="ref14.html#r1338">1338</a><a name="r1338" id="r1338"></a>]. Molecular assembly is driven and steered by maximizing fulfilled hydrogen bonding and reducing the number of unmet hydrogen bonding opportunities (so-called 'frustration') at the protein-water interface [<a title="go to cited reference" href="ref29.html#r2843">2843</a><a name="r2843" id="r2843"></a>].
      Strong water&#8208;water  enthalpic attractions are balanced against  the water molecule's high mobility in entropic terms. The complete arrest of a water molecule's mobility sets a limit on the entropic contribution of a water displacement process, while the solvent environment sets limits on ligand reactivity  [<a title="go to cited reference" href="ref39.html#r3813">3813</a><a name="r3813" id="r3813"></a>].</p>
      <p>&nbsp;</p>
      <p align="left">Generally, hydrophilic residues make up the protein&minus;protein and protein&minus;DNA binding surfaces. These interact via water molecules. The relative diameters of the binding cavity and the binding ligand determine the kinetics and thermodynamics of  (hydrophobic) binding cavity-ligand systems. This binding involves a sharp (cooperative) dewetting transition in their formation and a continuous dewetting transition on the loss of the  ligand [<a title="go to cited reference" href="ref26.html#r2575">2575</a><a name="r2575" id="r2575"></a>]. The binding energy has been attributed to changes in the free energy of the networks of hydrogen bonds that are formed, broken, and re-arranged when two hydrophobic surfaces approach (but do not necessarily contact) one another[<a title="go to cited reference"  href="ref27.html#r2670">2670</a><a name="r2670" id="r2670"></a>], and this may be viewed as the hydrophobic effect. Removal of water from the binding site (before ligand binding) may be assisted by nonpolar ligand groups due to a reduction in the interference of the previously bound water with the protein's internal hydrogen bonding and increased bulk hydrogen-bonding [<a title="go to cited reference"  href="ref30.html#r2922">2922</a><a name="r2922" id="r2922"></a>].</p>
      <p>&nbsp;</p>
      <p align="left">The water at the interface of the tightly bound barnase-barnstar enzyme-inhibitor complex (see right) shows highly slowed dynamics (&gt; 100-fold, [<a title="go to cited reference" href="ref28.html#r2797">2797</a><a name="r2797" id="r2797"></a>], from 20 bridging and 74 interfacial water molecules compared with 459 non-interfacial water molecules, reminiscent of glassy water even at ambient temperatures. Similar effects were noted at protein&minus;DNA interfaces. The strength of these effects is thought due to the  presence of a strong electrostatic field, driven by the binding surface&rsquo;s electrostatic complementarity [<a title="go to cited reference" href="ref28.html#r2797">2797</a>].</p>
      <p>&nbsp;</p>
      <p align="left"> Many  kinase inhibitors (used as cancer drugs) have recognition mechanisms involving  water molecules structured by the surrounding protein  [<a title="go to cited reference" href="ref23.html#r2277">2277</a><a name="r2277" id="r2277"></a>]. Water's small size, polarity, and conformational flexibility, together with the strength and directionality of the interactions, ensure good fits while retaining flexibility and ease of reversibility. The driving force for binding depends not only on the interaction of the biological molecules with each other but the energetic cost for the necessary removal of hydration water and the energetic gain for the subsequent molecular rearrangement of the hydration water molecules [<a title="go to cited reference" href="ref18.html#r1793">1793</a><a name="r1793" id="r1793"></a>, <a title="go to cited reference" href="ref19.html#r1805">1805</a><a name="r1805" id="r1805"></a>, <a  href="ref24.html#r2363" name="r2363" title="go to cited reference" id="r2363">2363</a>]. It may well be that the main driving force for the protein-protein interactions is not newly formed electrostatic attractive interactions between the proteins but the entropic gain by the displaced water, as shown for actin-myosin binding [<a  href="ref27.html#r2614" name="r2614" title="go to cited reference" id="r2614">2614</a>]. The use of water may also be useful in broadening the specificity of such links; for example, the peptide-binding  protein OppA uses several flexibly-adaptive water molecules to hydrogen bond  and shield charges when binding to lysine-X-lysine tripeptides, where  X is any one of the twenty common amino acids [<a title="go to cited reference" href="ref15.html#r1445">1445</a><a name="r1445" id="r1445"></a>].  
        In some cases, structured and structuring water molecules have also been found to be very important for inhibitor recognition [<a title="go to cited reference" href="ref22.html#r2128">2128</a><a name="r2128" id="r2128"></a>]. </p>
      <p align="left">&nbsp;</p>
      <p align="left">Water may have a significant role in ligand binding even when it may not be seen in the crystals to be directly involved. For example, the binding of sialic acid to the <em>Haemophilus influenzae</em> virulence protein depends on the placement of a water molecule influenced by a local alanine residue. The ligand is not bound when this water molecule is displaced by an asparagine mutant. It was suggested that the solvent structure operates as an evolutionary constraint on the protein sequence that contributes to ligand affinity and selectivity [<a title="go to cited reference" href="ref38.html#r3771">3771</a><a name="r3771" id="r3771"></a>].  Water molecules  stabilize <em>Escherichia coli</em> L-arginine/agmatine transporter AdiC and act as placeholders for atoms of the AdiC substrates L-arginine and agmatine [<a title="go to cited reference" href="ref44.html#r4313">4313</a><a name="r4313" id="r4313"></a>].</p>
      <p>&nbsp;</p>
      <p align="right"> [<a title="go to top of page" href="#top"><span class="style16">Back to Top&nbsp;</span><img src="images/rtup.gif" border="0" alt="to top of page" width="13" height="12" /></a>] </p>
</div>
    <h2>
<figure  class="floatright">
<figcaption>
      <p align="center"> Molecular recognition through hydrogen-bonding</p>
      <p align="center">&nbsp;</p>
</figcaption>
<img class="floatright" src="images/ligand.gif" width="245" height="118" alt="Hydrogen bonding through water enabling molecular recognition" title="Hydrogen bonding through water enabling molecular recognition" /> </figure>
<a name="func" id="func"></a>Water in protein and enzyme function</h2>
    <p>&nbsp; </p>
    <p>The energetic optimization of mutual 
      hydrogen-bonded networks between protein, water, and 
      ligand is an intrinsic part of the molecular recognition 
      process in enzymes, binding proteins and biological 
      macromolecules generally [<a title="go to cited reference" href="ref5.html#r412">412</a><a name="r412" id="r412"></a>]. Note that water-bound and oriented in empty ligand 
      binding sites will reduce the activation 
      entropy  when replaced by the ligand [<a title="go to cited reference" href="ref5.html#r414">414</a><a name="r414" id="r414"></a>]. Similarly, water next to protein surfaces involved in complex formation also shows reduced motion (lower entropy). In contrast,areas  that do not contribute to protein-protein complex formation show increased motion, thus suggesting that evolution of protein surfaces maximizes the entropy gain arising from the exclusion of hydration water when forming dry protein-protein interfaces  [<a title="go to cited reference" href="ref25.html#r2428">2428</a><a name="r2428" id="r2428"></a>]. An estimate of the entropy lowering of biological water relative to bulk water is 1.7 KJ  &#739; mol<sup>&minus;1</sup>  [<a title="go to cited reference" href="ref43.html#r4234">4234</a><a name="r4234" id="r4234"></a>].</p>
    <p>&nbsp;</p>
    <p>Proteins may create <a title="go for more information"  href="aqueous_biphasic.html">aqueous biphasic systems</a> at their interfaces, which can be used to partition materials into, or out of particular water pools. Many membrane proteins, such as the translocons required for the cotranslational assembly of membrane proteins, contain large water-filled cavities (water pools).  For example, the heterotrimeric SecY translocon complex partitions   membrane protein molecules by creating a water pool  that conducts nascent hydrophobic peptide chains through the translocon during secretion or membrane insertion. These  pools contain water that behaves very differently from bulk water, having retarded rotational dynamics and aligned dipoles [<a title="go to cited reference" href="ref25.html#r2446">2446</a><a name="r2446" id="r2446"></a>]. </p>
    <p>&nbsp;</p>
    <p>Water molecules are arranged/arrange themselves, forming a gradient in water dynamics around 'active' sites such that they facilitate the reactions via  the active site's 'hydration funnel' [<a title="go to cited reference"  href="ref23.html#r2237">2237</a><a name="r2237" id="r2237"></a>, <a title="go to cited reference"  href="ref25.html#r2475">2475</a><a name="r2475" id="r2475"></a>]. This arrangement often leads to the localized water molecules being activated for reaction either directly or catalytically. Also, it may lead to small empty spaces (the so-called dehydrons) and the fundamental  reactivity of surface water [<a title="go to cited reference" href="ref22.html#r2164">2164</a><a name="r2164" id="r2164"></a>].</p>
    <div style="float:none; clear:both;" align="center">
      <div align="center">
        <p>&nbsp;</p>
        <p><span class="style15"><a name="Fig" id="Fig">A chain of ten water molecules linking the end of one helix to the middle of another in glucoamylase-471</a></span></p>
        <p>&nbsp;</p>
        <p><img border="0" src="images/protein_hydration.gif" title="A chain of ten water molecules linking the end of one helix to the middle of another in glucoamylase-471" alt="A chain of ten water molecules linking the end of one helix to the middle of another in glucoamylase-471" width="480" height="315" /></p> 
      </div>
      <div style="width:90%; padding-left:5%; padding-right:5%">
        <p align="left"><span class="style3"><strong>Figure 1. </strong>The  water network links secondary structures within the protein, determining the  fine detail of the protein's structure and how particular molecular vibrations  may be preferred.  The above chain of ten 
            water molecules, linking the end of one &alpha;-helix&nbsp; 
            (helix 9, 211-227) to the middle of another (helix 
            11, 272-285) is found from the X-ray diffraction data 
            of glucoamylase-471, a natural proteolytic fragment 
            of <em>Aspergillus awamori</em> glucoamylase [<a title="go to cited reference" href="ref2.html#r155">155</a>].</span> </p>
        <p>&nbsp;</p>
</div>
</div>
    <div style=" float:none; clear:both;">
      <div class="floatright" style="width:400px;"> 
<figure  class="floatright">
<figcaption>
      <p align="center">Conserved water-binding site in ribonucleases</p>
      <p align="center">&nbsp;</p>
</figcaption>
<img border="0" src="images/ribonuclease.gif" title="Conserved water binding site in ribonucleases" alt="Conserved water binding site in ribonucleases" width="400" height="258" /></figure>
        <p align="left"><span class="style3"><strong>Figure 2.</strong> The above centrally-placed water molecule makes strong 
            hydrogen bonds to residues in three separated parts 
            of the ribonuclease molecule, holding them together. 
            This water molecule and its binding site are conserved 
            across the entire family of microbial ribonucleases 
            [<a href="ref4.html#r345" name="r345" title="go to cited reference" id="r345">345</a>].</span></p>
</div>
      <p>Water molecules have also proved integral 
        to the <a href="http://www.umassmed.edu/bmp/faculty/royer.cfm?start=Figures" title="opens new site in a pop-up window"   target="otherWin12" onclick="newWin('otherWin12')">structure 
        and biological function of a dimeric hemoglobin</a> [<a href="ref4.html#r377" name="r377" title="go to cited reference" id="r377">377</a>, <a href="ref41.html#r4078" name="r4078" title="go to cited reference" id="r4078">4078</a>], where the dynamic<br />
        and entropic changes upon ligand binding are coupled to the protein&ndash;water contact dynamics. Hydration waters are trapped deep inside the core region of an A<sub>2A</sub> G-protein coupled adenosine receptor  enabling  an allosteric network of water-mediated inter-residue contacts between the ligand-binding pocket to the G-protein binding site. This allosteric network plays a &nbsp;crucial role in the receptor regulation [<a href="ref41.html#r4078" title="go to cited reference" >4078</a>].</p>
      <p>&nbsp;</p>
      <p>The 
        internal water molecules in proteins have been surveyed 
        [<a title="go to cited reference" href="ref8.html#r725">725</a><a name="r725" id="r725"></a>]. They stabilize proteins by filling internal cavities and  interacting with backbone residues of loops and other polar atoms buried in the protein core. They may also act as lubricants to favor loop dynamics 
        [<a title="go to cited reference" href="ref23.html#r2296">2296</a><a name="r2296" id="r2296"></a>]. Many enzyme processes utilize several domains or subunits that fluctuate between several states during the catalysis. These fluctuations involve only low-energy structural changes due to water's  directing and lubricating action [<a title="go to cited reference" href="ref29.html#r2855">2855</a><a name="r2855" id="r2855"></a>].</p>
      <p>&nbsp;</p>
      <p>The energy barrier in proton-transfer and electron-transfer 
        enzymic reactions may be lowered by creating a  large polarizability at the active site due to electrostatic fluctuations of the  hydration water [<a title="go to cited reference" href="ref30.html#r2964">2964</a><a name="r2964" id="r2964"></a>]. Both proton-transfer processes [<a title="go to cited reference" href="ref10.html#r907">907</a><a name="r907" id="r907"></a>, <a title="go to cited reference" href="ref2.html#r160">160b</a><a name="r160" id="r160"></a>] and electron-transfer 
        reactions [<a title="go to cited reference" href="ref10.html#r908">908</a><a name="r908" id="r908"></a>] 
        may be further facilitated by ordered (hydrogen-bonded) water molecules connecting proton 
        donor to acceptor sites or electron donor to electron acceptor 
        sites, respectively [<a  href="ref43.html#r4252" name="r4252" title="go to cited reference" id="r4252">4252</a>]. In both cases, the transfer is faster 
        where the linking water molecules possess stronger hydrogen 
        bonding. Low-barrier hydrogen bonds form part of proton wires when the p<em>K</em><sub>a</sub> values of neighboring hydrogen-bond donor and acceptor amino acids are nearly equal. These may facilitate both directions of an  electron transfer pathway, as in the water oxidation in photosystem II (PSII, protein data bank code: 3ARC)  [<a title="go to cited reference" href="ref41.html#r4093">4093</a><a name="r4093" id="r4093"></a>]. With the proton on its histidine (D1-His190) (also hydrogen-bonded to the carbonyl of an asparagine, D1-Asn298), the redox potential of its neighboring tyrosine (D1-Tyr161) is the lowest and can serve as an electron donor directly to the oxidized chlorophyll.  However, with the proton on the tyrosine (D1-Tyr161), the redox potential of both the tyrosine (D1-Tyr161) and   histidine (D1-His190) are equal  and can serve as an electron donor to PSII's manganese center (Mn<sub>4</sub>CaO<sub>5</sub>) via a <a href="hydrogen_bonding.html#wire">water wire</a>. Another water wire allows the transfer of the proton from the catalytic site  to the protein bulk surface   [<a title="go to cited reference" href="ref42.html#r4105">4105</a><a name="r4105" id="r4105"></a>].</p>
      <p>&nbsp;</p>
      <p>The capability to transport of water molecules or protons down the water chains (wires) depends on the surrounding amino acids.  If the water wire is surrounded by hydrophobic side-chains or hydrogen-bond donors such as the amino groups of asparagine and glutamine,  it can only transport water. If the water wire is surrounded by H-bond acceptors such as aspartic acid, glutamic acid, and backbone carbonyl groups, it is capable of proton transport [<a title="go to cited reference" href="ref40.html#r3999">3999</a><a name="r3999" id="r3999"></a>]. In the photosynthetic protein photosystem II (PSII), a water-splitting reaction is driven by light,</p>
      <p>&nbsp;</p>
      <p align="center">2 H<sub>2</sub>O  <img src="images/reg.gif" alt="-&gt;" width="22" height="10" /> 4 H<sup>+</sup> + O<sub>2</sub> + 4 e<sup>&minus;</sup></p>
      <p><br />
      where PSII incorporates two water molecules and releases four protons. Here, the water chain is surrounded by hydrogen-bond acceptors, and the water molecules are rigidly fixed, so the water chain transports the protons rather than the water molecules. In aquaporin, the water chain is surrounded by hydrophobic side-chains and hydrogen-bond donors, and where the water molecules are flexible and fluctuating, so the water wire transports water molecules at up to three water molecules per nanosecond but does not transport protons.</p>
      <p>&nbsp;</p>
      <p>In the cyanobacterial photosystem II, a water channel has been found linking bulk water to the active Mn<sub>4</sub> CaO<sub>5</sub> cluster and critical hydrogen-bond water networks within the protein's interior, where the supplied water molecules are oxidized [<a title="go to cited reference" href="ref30.html#r2979">2979</a><a name="r2979" id="r2979"></a>].  In redox metalloenzymes, including those in respiration, energy conversion, and photosynthesis,  electron transfer is often accompanied by proton movement (proton-coupled electron transfer, PCET)  [<a title="go to cited reference" href="ref40.html#r3910">3910</a><a name="r3910" id="r3910"></a>]. Such processes almost invariably involve water molecules.</p>
      <p>&nbsp;</p>
      <p  align="center" class="style15">Peroxide bond cleavage in ascorbate peroxidase leading to the formation of compound <span class="style1">I</span>, derived from [<a title="go to cited reference" href="ref40.html#r3910">3910</a>]</p>
      <p  align="center" class="style15">&nbsp;</p>
      <p align="center"><img src="images/redoxmech.gif" width="720" height="169" alt="Peroxide bond cleavage in ascorbate peroxidase leading to the formation of compound I, derived from [3910]"  title="Peroxide bond cleavage in ascorbate peroxidase leading to the formation of compound I, derived from [3910]"/></p>
      <p align="center">&nbsp;</p>
      <figure  class="floatleft">
  <figcaption>
      <p align="center">&nbsp;</p>
      <p align="center">Ascorbate &rarr;&nbsp;Monodehydroascorbate radical + e-</p>
      <p align="center">&nbsp;</p>
	    </figcaption><img src="images/vitc.gif" width="320" height="95" alt="Ascorbate Monodehydroascorbate radical" title="Ascorbate Monodehydroascorbate radical"/></figure>
         <p>The ascorbate peroxidase then  converts ascorbate to the mono-dehydroascorbate radical  with one-electron oxidations.      </p>
      <p align="center">&nbsp;</p>
      <p align="center">Compound I + Ascorbate &rarr; Compound II + monodehydroascorbate radical &bull;</p>
      <p align="center"><br />
        Compound II + Ascorbate  &rarr; Ascorbate peroxidase + monodehydroascorbate radical &bull; + H<sub>2</sub>O      </p>
      <p>&nbsp;</p>
      <figure  class="floatright">
  <figcaption>
      <p align="center">Putative water wire</p>
      <p align="center">&nbsp;</p>
  </figcaption>
<img  class="floatleft" src="images/wwire.gif" title="putative water wire" alt="putative water wire" width="200" height="120" /></figure>
     
      <p align="left"><a name="wire" id="wire"></a>Local water structures and proton-conducting hydrogen-bonded water wires (see right) can form rapidly (&asymp; 10 ns or so) in response to control by relatively small-scale  rearrangements of the protein matrix [<a title="go to cited reference" href="ref22.html#r2101">2101</a><a name="r2101" id="r2101"></a>].  In some cases, the water wire may be affected by the binding of different ions. An example is the Cl<sup>&minus;</sup> /H<sup>+</sup> transporter protein that catalyzes the 2:1 stoichiometrically coupled exchange of Cl<sup>&minus;</sup> and H<sup>+</sup> across biological membranes. The water wire is positioned with the aid of the chloride ion, but other ions such as F<sup>&minus;</sup>, NO<sub>3</sub><sup>&minus;</sup>, and SCN<sup>&minus;</sup> bind differently and change the stoichiometry or destroy the active wire  [<a title="go to cited reference" href="ref26.html#r2576">2576</a><a name="r2576" id="r2576"></a>]. The movements of protons through proton-transport membrane channels are coupled to the diffusion of water through the pores, but these protons cross faster than the diffusion of H<sub>3</sub>O<sup>+</sup>. As examples, both the voltage-gated proton channel 1 protein (Hv1) [<a title="go to cited reference" href="ref29.html#r2849">2849</a><a name="r2849" id="r2849"></a>], implicated in diseases including cancer, and the influenza M2 channel  [<a title="go to cited reference" href="ref31.html#r3085">3085</a><a name="r3085" id="r3085"></a>],  essential for the reproduction of the flu virus, are trans-membrane four-helix bundles that transport protons down water wires. </p>
      <p align="left">&nbsp;</p>
       <figure  class="floatright">
<figcaption>
      <p align="center">Water-assisted asparagine recognition and</p>
      <p align="center"> aspartate discrimination by asparaginyl-tRNA synthetase</p>
      <p align="center">&nbsp;</p>
</figcaption>
<img class="floatleft" src="images/roles-3.gif" width="480" height="233" alt="Cartoon showing water-assisted asparagine recognition and aspartate discrimination by asparaginyl-tRNA synthetase" title="Cartoon showing water-assisted asparagine recognition and aspartate discrimination by asparaginyl-tRNA synthetase"/></figure>
<p align="left">&nbsp;</p>
    
      <p><a name="fig3" id="fig3"></a></p>
      <p><strong>Figure 3.</strong> Cartoon showing water-assisted asparagine recognition and aspartate  discrimination by asparaginyl-tRNA synthetase. (a) &nbsp;Amongst the bound water molecules in the  unliganded enzyme, one (Wat1) is bound between the Leu<sub>229</sub> peptide carbonyl and the  Glu<sub>230</sub> carboxylate and another (Wat4) between the side chains of Tyr<sub>333</sub> and  Arg<sub>364</sub>. (b) These water molecules link to the amide group on the asparagine-AMP  substrate on binding, and two water molecules are released (Wat2 and Wat3).  Aspartate-AMP cannot bind at the same site as the Wat1 water molecule is not  able to donate three hydrogen bonds. Figure partially redrawn from [<a title="go to cited reference" href="ref24.html#r2336">2336</a><a name="r2336" id="r2336"></a>].</p>
      <span class="style3"></span>
      <p>&nbsp;    </p>
      <p align="left"><a name="ba3" id="ba3"></a>The heme-copper <em>ba<sub>3</sub></em>-oxidoreductase from <em>Thermus thermophilus</em> uses a highly conserved water molecule (w941), held between two propionate groups as part of a one proton input channel, to deliver protons to the active site for both O<sub>2</sub> chemistry and proton pumping (see below, [<a title="go to cited reference" href="ref24.html#r2322">2322</a><a name="r2322" id="r2322"></a>]); w = water, H = histidine, D = aspartate, prop = propionate, Y = tyrosine).</p>
      <p align="left">&nbsp;</p>
      <p align="center"><span class="style15">ba3-Cytochrome c oxidase, from <em>Thermus thermophilus</em>, 1EHK</span><img src="images/ba3_cytochrome.gif" alt="ba3-cytochrome c oxidase , from T. thermophilus, 1EHK" title="ba3-cytochrome c oxidase , from T. thermophilus, 1EHK" width="750" height="323" /></p>
<p align="left"> Several water molecules  are hydrogen-bonded to the propionate-A&ndash;H<sup>+</sup> from the heme-a<em><sub>3</sub></em>, and  w941 and w946 (as shown above) act as a <a href="hydrogen_ions.html#h5o2">Zundel cation</a> when protonated. The neighboring histidine (H376, near the 'free' water pool) controls the protonation state of the propionic acid (A) aspartic acid (D372) pair, and hence through another histidine (H283) to the heme-a<em><sub>3</sub></em> oxidation state [<a title="go to cited reference" href="ref24.html#r2322">2322</a>].     </p>
<figure  class="floatright">
  <figcaption>
      <p align="center">The catalytic zinc ion of a membrane type I matrix metalloproteinase </p>
      <p align="center">with a small peptide product (R'), from   [<a title="go to cited reference" href="ref31.html#r3058">3058</a>]</p>
      <p align="center">&nbsp;</p>
</figcaption>
<img src="images/mmp.gif" alt="The catalytic zinc ion of a membrane type I matrix metalloproteinase with a small peptide product (R'), from [3058]"  title="The catalytic zinc ion of a membrane type I matrix metalloproteinase with a small peptide product (R'), from [3058]" width="360" height="219" class="floatright" /></figure>
      <p align="left"><a name="mmp" id="mmp"></a></p>
      <p align="left">Water molecules can also be used by enzymes to facilitate the movement of substrates and products towards and away from their active sites. Shown right is the catalytic zinc ion  of a membrane type I matrix metalloproteinase with a small peptide  product  (R') formed immediately after peptide cleavage  [<a title="go to cited reference" href="ref31.html#r3058">3058</a><a name="r3058" id="r3058"></a>]. There are two crucial auxiliary water molecules at this active site. One water molecule (W<sub>B</sub>, situated in front) acts to stabilize the protonated reaction intermediate by binding to the &beta;-oxygen of the product carboxylate, while the second water molecule (W<sub>A</sub>,  situated behind) facilitates the product release by  hydrogen-bonding to between the &alpha;-oxygen of the product carboxylate and the glutamate carboxylate. </p>
      <p align="left">&nbsp;</p>
      <p align="left">High-resolution X-ray structures of proteins  show the presence of water ions inside and on the surface of proteins from their close oxygen-oxygen distances, typical of the <a href="hydrogen_ions.html#h5o2">Zundel ions</a> [<a title="go to cited reference" href="ref39.html#r3822">3822</a><a name="r3822" id="3822"></a>]. </p>
      <p align="right">[<a title="go to top of page" href="#top"><span class="style16">Back to Top&nbsp;</span><img src="images/rtup.gif" border="0" alt="to top of page" width="13" height="12" /></a>]      </p>
      <center>
        <h2><a name="bip" id="bip"></a>Enzymic reactions in biphasic liquid systems</h2>
      </center>
      <p>
        It would often be useful if enzyme-catalyzed reactions could be performed in solvents other than water, as this is
        not the ideal medium for most organic reactions <font face="arial">[<a title="go to cited reference" href="ref42.html#r4186">4186</a><a name="r4186" id="r4186"></a>]</font>. Many reactants (e.g., molecular oxygen, steroids, and lipids) are more soluble in organic solvents than
        in water, and some products may be relatively labile in an aqueous environment.
        Accomplishing reactions with such substrates (e.g., the aerobic oxidation of
        estrogens catalyzed by fungal laccase) in non-aqueous media allows a much
        increased volumetric activity to be achieved. Microbial contamination, by
        contrast, is much less of a problem in such solvents, and the consequent absence
        of microbial proteases may lead to an apparent stabilization in the biocatalyst.</font></p>
      <p>&nbsp;</p>
      <p><font face="Arial">Some polymerizing reactions, for example, the polymerization of
        phenols catalyzed by peroxidase, will produce a higher molecular weight product
        when carried out in a solution more able to dissolve the products (i.e., oligomers) initially formed. Under normal physiological conditions, hydrolytic
        enzymes catalyze the degradation of polymers; i.e., hydrolases are transferases
        normally transferring a moiety to the acceptor, water. Water is ordinarily   present
        in  vast molar excess over other potential acceptor molecules, so no reaction
        occurs other than hydrolysis. Also, the normal 'concentration' of water (~
        55.5 M) is much greater than its typical K<sub>m</sub> (~ 50 mM), and the rate
        of hydrolysis will not be affected as the reaction proceeds. By significantly reducing
        the <a href="water_activity.html">water activity</a> in these systems, they can be used to transfer to other
        acceptors. Examples of this can be found in the transesterification reactions of
        esterases and lipases and the (undesirable)
        formation of isomaltose from glucose catalyzed by glucoamylase.</font> Strengthening the protein surface hydration via surface charge engineering has proven an effective and efficient rational strategy for tailoring enzyme stability in organic solvents <font face="arial">[<a title="go to cited reference" href="ref42.html#r4185">4185</a><a name="r4185" id="r4185"></a>]</font>.</p>
      <p>&nbsp;</p>
      <p><font face="Arial">Restriction of
        the enzyme to the aqueous phase effectively immobilities the enzyme and allows
        its straightforward separation, using phase separators developed for the
        established chemical process industry from the product-containing organic phase.
        hHowever, the main asset of these systems is their ability to shift the
        thermodynamic equilibria of the reactions </font><font face="arial">[<a title="go to cited reference" href="ref42.html#r4188">4188</a><a name="r4188" id="r4188"></a>].</font><font face="Arial"> Enzymes do not change the equilibrium constants (K<sub>eq</sub>) of reactions.Although  changes in the physical conditions (i.e., temperature, pH, and pressure) affect  the K<sub>eq</sub> of a reaction, this effect is usually relatively slight over the  physical range allowed by the stability of the biocatalysts. However,  the use of
        a biphasic aqueous-organic system may result in substantial changes in
        the practically useful 'apparent' K<sub>eq</sub>. The use of enzymes within
        organic solvents normally results in a two-phase system as all water-soluble
        enzymes possess a significant amount of strongly bound water, even when in an
        apparently dry state. </font></p>
      <p>&nbsp;</p>
     <figure  class="floatright">
      <img src="images/fig7_1.gif" width="480" height="378" alt=""/></figure>
      <p align="left"><font face="arial">Right shows a schematic diagram showing two
        configurations for an enzyme within an organic solvent. (a) almost-anhydrous
        enzyme suspended in the organic solvent. The enzyme (E) is surrounded by a thin
        interphase consisting of water or water plus immobilization support. (b) enzyme
        dissolved in a reversed micellar medium. The micelles are formed by the
        surfactant molecules with assistance from any cosurfactant (if present). The
        surfactant (e.g., cetyltrimethylammonium bromide (CTAB), bis(2-ethylhexyl) sodium
        sulphosuccinate (AOT), phosphatidylcholine, tetraethyleneglycol-dodecylether)
        is only found at the interphase boundary. In contrast, the water-immiscible
        cosurfactant (e.g., butanol, hexanol, octanol), added to vary the properties of
        this interphase, is generally less polar and more soluble in the continuous organic
        phase. Both preparations (a) and (b) give optically transparent
      solutions.</font></p>
      <p align="right">&nbsp;</p>
      <center>
      </center>
      <p><font face="arial">It should become clear from the later discussion that there may
        be a substantial advantage to be gained from  using <a href="aqueous_biphasic.html">biphasic systems</a> in many
        enzyme-catalyzed reactions. One major factor must first be addressed; the
        stability of the enzyme in these systems. A distinction should be drawn between
        the more water-soluble hydrophilic enzymes and the more hydrophobic enzymes
        often associated with lipid and membranes (e.g., lipases). Hydrophilic  enzymes' functional integrity and stability appear to depend on a thin layer of  water, just a few molecules thick, within the microenvironment. This
        amount of water is minuscule (between 50 and 500 molecules of water for each
        enzyme molecule), and the enzyme may effectively be operating in an almost
        anhydrous state. Some hydrophobic lipases retain activity even if fewer
        water molecules remain, presumably just sufficient to stabilize the
        conformation of the active site. The pH of such minute pools of water,
        containing no free hydrogen ions, is impossible to measure or control 
        directly. However, it appears that the enzyme 'remembers' the pH of its last
        aqueous solution and functions as though still at that pH. If the enzyme-bound water
        is stripped out or diluted by  using more water-soluble, or miscible,
        organic solvents, then the enzyme is usually inactivated. However, under
        conditions where this does not occur, the limited amount of water available, and
        the associated reduction in the water activity, considerably reduce the rate of
        thermoinactivation. This has a stabilizing effect on most enzymes; porcine
        pancreatic lipase, for example, has a half-life of greater than 12 hours at 100 &deg;C in 0.02% water in tributyrin, whereas this drops to 12 minutes
        at a 0.8% water content and inactivation is almost instantaneous in 100% water.
        Additionally, the freezing point of the water is reduced, which allows the use of
        particularly heat-labile enzymes at very low temperatures. The lowering of the
        water activity tends to produce a more rigid enzyme molecule which may affect
        both its K<sub>m</sub> and V<sub>max</sub>. In extreme cases, this may result in a
        change in the catalytic properties. Porcine pancreatic lipase demonstrates this
        effect. When used in biphasic systems of low water activity, it no longer
        catalyzes transesterification reactions involving the bulky tertiary alcohols. </font><font face="arial">The  solvent polarity is the most critical factor in balancing stabilization and  inactivation due to the organic phase.  Solvents of lower
        polarity (i.e., greater hydrophobicity) are less able to disrupt the structure of
        the necessary tightly bound water molecules. The best measure of polarity is the
          <a href="pure_water.html#phobic">logarithm of the partition coefficient </a>(Log<b><i>P</i></b>) of the organic
        liquid between n-octanol and water; the higher the Log<b><i>P</i></b>, the
      more nonpolar (hydrophobic) is the solvent.</font></p>
     
      <p>&nbsp;</p>
   
      <figure  class="floatright" width="405">
        <figcaption>
      <p align="left">&nbsp;&nbsp;Schematic diagram showing the dependence
        of the activity of the </p>
      <p align="left">&nbsp;&nbsp;immobilized enzymes, in biphasic systems, on the        Log<b><i>P</i></b> of the </p>
      <p align="left">&nbsp;&nbsp;organic phase. <font color="#FF0000"> ________</font> free enzyme;<b><font color="#FF0000">--------</font></b> enzyme 
        immobilized </p>
      <p align="left">&nbsp;&nbsp;on a strongly hydrophilic support</p>
      <p align="center">&nbsp;</p>
     
        </figcaption><img src="images/fig7_2.gif" width="400" height="305" alt="Schematic diagram showing the dependence of the activity

of the immobilised enzymes, in biphasic systems, on the 

LogP of the organic phase" title="Schematic diagram showing the dependence of the activity

of the immobilised enzymes, in biphasic systems, on the 

LogP of the organic phase"  class="floatright"/></figure>
      <p><font face="arial">There appears to be
        a clear correlation between the activity of biocatalysts in two-phase systems
        and the Log<b><i>P</i></b>. The S-shape of this relationship
        suggests that enzymes are generally inactivated by solvents with
        log<b>P </b>&lt; 2 but are little affected by solvents with
        Log<b><i>P</i></b> &gt; 4. There is some variation in the effects between
        different enzymes and different solvents, which makes activity prediction, in the
        Log<b><i>P</i></b> range of 2 - 4, somewhat  imprecise. This range includes some
        of the most utilized organic phases (e.g., chloroform), which may be suitable for
        some applications but cause harmful inactivation in others. The solubility of
        the reactant(s) and product(s) may considerably reduce the range of
        Log<b><i>P</i></b> available for a particular application; many
        basically nonpolar molecules possessing some polar structural regions which
        cause their lack of solubility in strongly hydrophobic solvents. The choice of
        organic phase will also depend on additional factors such as cost, ease of
        recovery, fire and fume hazards, and specific inhibitory effects. The S-shaped
        curve can be shifted to the left by immobilizing the enzyme within a highly
        hydrophilic support. A simple way of achieving this is to
        impregnate a beaded hydrophilic polymer (e.g., Sephadex, agarose) with the enzyme
        followed by suspension of the wet beads directly in the organic phase. These
        shifts are very important as they greatly increase the choice of suitable
        organic phases. Such impregnated beads have the additional advantages that </font></p>
      <p>&nbsp;</p>
      <ol type="a">
        <li><font face="arial">they protect the contained enzyme from liquid-liquid interfacial denaturation at
          higher rates of stirring; </font></li>
        <li><font face="arial">they enable more facile recovery of the
          biocatalyst; </font></li>
        <li><font face="arial">they may be used with low molecular weight hydrophilic
          coenzymes (e.g., NAD(P)<sup>+</sup>) with the assistance of a coenzyme regenerating
          process; the co enzyme being effectively immobilized within the aqueous pools;
          and </font></li>
        <li><font face="arial">they allow the efficient and continuous use of biphasic packed bed reactors so long as
          the moving organic phase remains saturated with water.</font></li>
      </ol>
      <p>&nbsp;</p>
      <p><font face="arial">Biphasic systems may be further stabilized by the use of deuterated
        water (D<sub>2</sub>O). This reduces the rate of thermal inactivation, although
        it does cause an increase in the p<em>K</em><sub>a</sub>s of ionizing groups by about 0.4
        with the associated changes in the pH-activity and pH-stability relationships.
        The higher cost of the deuterated water is offset, to a certain extent, by the
        small amount necessary in these systems and the ease with which it may be
        recovered at the end of a catalytic process. </font></p>
      <p>&nbsp;</p>
      <p><font face="arial">Even where the organic solvent has
        very low Log<b><i>P</i></b> and is miscible, the effect of the expected loss
        in enzymic activity may be offset by changes in the equilibrium constant. Thus,
        it has been proposed that glucose isomerase be used in aqueous ethanol to
        produce high fructose corn syrup. The equilibrium fraction of fructose can be
        raised by about 10% to 55% at 30&deg;C in 80% v/v ethanol   [<a title="go to cited reference" href="ref42.html#r4187">4187</a><a name="r4187" id="r4187"></a>]. This is
        economically valid even though the enzymic activity drops by about 10% compared
        to that in the absence of ethanol. </font></p>
<p align="right">[<a title="go to top of page" href="#top"><span class="style16">Back to Top&nbsp;</span><img src="images/rtup.gif" border="0" alt="to top of page" width="13" height="12" /></a>]      </p>
      <div style=" float:none; clear:both;"></div>
<figure  class="floatright">
        <figcaption>
          <p align="center">&nbsp;</p>
          <p align="center">&nbsp;</p>
          <p align="center"><em>Tenebrio molitor</em> antifreeze protein, </p>
          <p align="center">from PDB  1EZG, binding to ice</p>
          <p align="center">&nbsp;</p>
        </figcaption>
        <img src="images/AFP.gif" width="320" height="364"  class="floatright" alt="Tenebrio molitor antifreeze protein, from PDB  1EZG, binding to ice" lt="Tenebrio molitor antifreeze protein, from PDB  1EZG, binding to ice" /></figure>
      <h2><a name="AFP" id="AFP"></a>Antifreeze proteins </h2>
      <p>Although most protein molecules do not adsorb onto ice, both ice-nucleating and antifreeze proteins (AFP)  [<a title="go to cited reference" href="ref39.html#r3815">3815</a><a name="r3815" id="r3815"></a>] bind to ice crystals. The reason for this difference concerns the interaction area; ice-nucleating proteins  form large aggregates, while antifreeze proteins are typically small and soluble in water [<a title="go to cited reference" href="ref37.html#r3602">3602</a><a name="r3602" id="r3602"></a>]. Evolutionary analysis reveals that antifreeze proteins do not emerge from a common ancestor; rather, their evolution is independent and they are active by an irreversible adsorption-inhibition mechanism [<a title="go to cited reference" href="ref39.html#r3815">3815</a>]. The different classes of AFP have been described [<a title="go to cited reference" href="ref39.html#r3815">3815</a>]. Antifreeze proteins possess  ice-binding sites [<a title="go to cited reference" href="ref36.html#r3583">3583</a><a name="r3583" id="r3583"></a>] that are typically flat and relatively hydrophobic [<a title="go to cited reference" href="ref29.html#r2819">2819</a><a name="r2819" id="r2819"></a>]. They introduce greater order (lower entropy) in their hydration shell for their enthalpic binding of the surface water molecules  [<a title="go to cited reference" href="ref31.html#r3048">3048</a><a name="r3048" id="r3048"></a>]. This is most 
        energetically favorable where there is no pre-existing order 
        in the water that requires destruction, and the surface water has dangling O-H groups available for the crystalline hydrogen bonding. The distance between the protein and ice surfaces determines the water network density and whether the protein is attracted or repulsed [<a title="go to cited reference"  href="ref38.html#r3739">3739</a><a name="r3739" id="r3739"></a>]. Surfaces that raise the <a href="water_activity.html">water activity</a> above that of the bulk water are ideal for this purpose. There is a synergy between hydrophobic and hydrophilic groups where the hydrophilic groups anchor the ice-binding sites, and the hydrophobic groups reduce its flexibility  [<a title="go to cited reference"  href="ref36.html#r3592">3592</a><a name="r3592" id="r3592"></a>]. Strongl hydrogen-bonding surfaces (with low water activity) force surface water molecules into particular structures that have difficulty in fitting to the ice crystals. A lower water density at the ice-binding site paves the way to the protein binding to ice nuclei, while higher solvent density at non-ice-binding surfaces protects against ice growth [<a title="go to cited reference"  href="ref37.html#r3645">3645</a><a name="r3645" id="r3645"></a>].</p>
      <p>&nbsp;</p>
      <p>Once the ice-binding surface has bound an ice crystal, it is the property of the rest of the protein to maintain a liquid-like character in its hydration water,  preventing further ice-nucleation and growth on top of the bound antifreeze proteins [<a title="go to cited reference" href="ref40.html#r3957">3957</a><a name="r3957" id="r3957"></a>]. The ice-active repeats of the ice-nucleating protein (InaZ) from the ice-nucleating bacterium <em>Pseudomonas syringae</em> adopt a &beta;-helical structure in solution (and at water surfaces) that imposes structural ordering on the adjacent water network so increasing as the interface is cooled to temperatures close to the melting point of water [<a title="go to cited reference" href="ref42.html#r4194">4194</a><a name="r4194" id="r4194"></a>].</p>
      <p>&nbsp;</p>
      <p>Interestingly, the clathrate pentagonal structuring has been found to <a href="https://www.youtube.com/watch?feature=player_embedded&amp;v=KTBH3Wf_E3s"   title="go to YouTube"   target="otherWin6" onclick="newWin('otherWin6')">extensively surround the helices in the four-helix bundle </a> of the antifreeze protein from winter flounder (an alanine-rich &alpha;-helical type; AFP-I). This pentagonal structuring retards the formation of the harmful ice crystals in fish living in near-freezing water [<a title="go to cited reference" href="ref21.html#r2084">2084</a><a name="r2084" id="r2084"></a>]. Type II AFPs are cysteine-rich globular AFPs found in smelt and Arctic herring. Rather than binding directly with ice, the antifreeze globular protein type three (AFP-III), from the Antarctic eelpout, arranges some of its surface water molecules into an ice-like array  well above 0&nbsp;&deg;C but retained below 0&nbsp;&deg;C, which it then uses to bind to ice crystals  [<a title="go to cited reference" href="ref29.html#r2801">2801</a><a name="r2801" id="r2801"></a>]. AFP-III also adsorbs at the air-water interface   [<a title="go to cited reference" href="ref35.html#r3440">3440</a><a name="r3440" id="r3440"></a>]. The thermal hysteresis, ice recrystallization inhibition, and cryopreservation activity of three isoforms of the notched-fin eelpout AFP-III have been studied,  as well as the structural and dynamic features of  their mutants' ice-binding surfaces [<a title="go to cited reference" href="ref42.html#r4192">4192</a><a name="r4192" id="r4192"></a>]. In another protein, poly-pentagonal water networks are observed in a protein capable of binding to ice crystals. This protein has about fifty sem-clathrates pentagonal water networks with more than 400 surface waters forming a poly-pentagonal network with a perfect position match to the waters constructing the first prism and pyramidal ice planes [<a title="go to cited reference" href="ref40.html#r3947">3947</a><a name="r3947" id="r3947"></a>]. It is the precise positioning of the pentagonal clusters that is important for their antifreeze / ice-binding properties.</p>
      <p>&nbsp; </p>
      <p>In the <em>Tenebrio</em> beetle (insect) antifreeze protein, there are tandem 12-residue repeats (TCTxSxxCxxAx) that form  a &beta;-helix with regularly spaced threonine residues (0.744  nm and 0.464 nm), each turn of the helix, that make a match to water molecules in the ice prism plane  (0.738 nm and 0.452 nm (see right  [<a title="go to cited reference" href="ref29.html#r2802">2802</a><a name="r2802" id="r2802"></a>]). It has been found that ice recognition occurs by slow diffusion of the protein to present the proper orientation  to the ice surface, followed by a rapid collective organization of the hydration water at the ice-binding surface to form an anchored clathrate motif that latches the protein to the ice surface [<a href="ref34.html#r3397" name="r3397" title="go to cited reference" id="r3397">3397</a>]. </p>
      <p>&nbsp;</p>
      <p>A similar binding site has been found in the ice-binding protein of the Antarctic bacterium <em>Marinomonas primoryensis</em> (MpAFP) that is used to anchor the organism to Antarctic ice flows where oxygen and nutrients are more available [<a title="go to cited reference" href="ref29.html#r2805">2805</a><a name="r2805" id="r2805"></a>]). The protein binds to sea ice through a flat, repetitive two-dimensional array of Thr and Asn residues. Both polar and nonpolar groups create order in the water molecules 
        surrounding them, but their ability to do this and the types 
        of ordering produced are very different. Polar groups are 
        most capable of creating ordered hydration through <a title="go to 'Hydrogen bonding in water' page"  href="water_hydrogen_bonding.html">hydrogen 
          bonding</a> and ionic interactions (an excellent guide to <a href="http://www.biochem.ucl.ac.uk/bsm/atlas/" title="opens new site in a pop-up window"   target="otherWin12" onclick="newWin('otherWin12')">amino 
            acid hydrogen bonding</a> is given elsewhere). </p>
<p align="right">[<a title="go to top of page" href="#top"><span class="style16">Back to Top&nbsp;</span><img src="images/rtup.gif" border="0" alt="to top of page" width="13" height="12" /></a>] </p>

    <hr />
    <h2>Footnotes</h2>
    <p><a name="a" id="a"></a><sup>a</sup> To avoid such activity loss, 
        proteins generally avoid crystal formation, perhaps by evolutionary 
        design involving surface <a href="kosmotropes_chaotropes.html" title="more on kosmotropes and chaotropes">kosmotropic</a> lysine residues that minimize self-aggregation [<a title="go to cited reference" href="ref8.html#r771">771</a><a name="r771" id="r771"></a>]. 
        [<a title="click for an explanation" href="#ba"><span title="go back to the text">Back</span></a>]</p>
    <p>&nbsp;</p>
    <div style=" float:none; clear:both;"> 
<figure  class="floatright">
<figcaption>
      <p align="center">Hydrophobicity</p>
      <p align="center">&nbsp;</p>
</figcaption>
<img class="floatright" src="images/haa.gif" width="360" height="202" title="amino acid hydrophobicity/hydrophilicity" alt="Hydrophobicity: (phenylalanine=isoleucine=leucine)&gt;(tryptophan-cysteine=cystine=valine=methionine)&gt;tyrosine&gt;(alanine=histidine)&gt;(glycine=threonine=proline)&gt;(serine=glutamine=asparagine=arginine)&gt;(aspartic acid=glutamic acid)&gt;lysine" /></figure>
      <p>&nbsp;</p>
      <p><sup>b<a name="b" id="b"></a></sup> A historical review of the 'hydrophobic effect' in proteins is available [<a title="go to cited reference" href="ref14.html#r1393">1393</a><a name="r1393" id="r1393"></a>].</p>
      <p>&nbsp;</p>
      <p>An approximate 
          comparison of the hydrophobicity of the amino acids is 
          given in the table opposite. Further discussion of relative 
          hydrophobicities is <a href="http://www.mat.univie.ac.at/&asymp; neum/software/protein/aminoacids.html" title="opens new site in a pop-up window"   target="otherWin2" onclick="newWin('otherWin2')">given 
        elsewhere</a>, as is a classification of hydrophobicity 
          scales [<a title="go to cited reference" href="ref7.html#r633">633</a><a name="r633" id="r633"></a>]. 
          It should be noted that such hydrophobic interactions 
          are particularly important in stabilizing inter-domain 
          and quaternary interactions. Also, the protein surface may be made up of 25% -45% hydrophobic residues. </p>
      <p>&nbsp;</p>
      <p>The hydration shells of zwitterionic  amino acids have been quantitatively determined by Raman spectroscopy with multivariate curve resolution (Raman-MCR) and differential scanning calorimetry (DSC)  [<a name="r4379" id="r4379"></a><a title="go to cited reference"  href="ref44.html#r4379">4379</a>].      [<a title="click for an explanation" href="#bb"><span title="go back to the text">Back</span></a>]</p>
      <p>&nbsp;</p>
    </div>
    <div style=" float:none; clear:both;">
      <p><sup>c<a name="c" id="c"></a> </sup>The effect of strongly bound 
          surface water molecules on preventing protein-protein interactions 
          is described in [<a title="go to cited reference" href="ref9.html#r881">881</a><a name="r881" id="r881"></a>]. 
          [<a title="click for an explanation" href="#bc"><span title="go back to the text">Back</span></a>]</p>
</div>
    <p>&nbsp;</p>
    <p><sup>d<a name="d" id="d"></a> </sup>Alternatively to this molecular 
        group approach, some of this increased density may be attributed 
        to electrostriction pressure (local pressure 
        increase due to the localized electric field) [<a title="go to cited reference" href="ref10.html#r951">951</a><a name="r951" id="r951"></a>]. 
        [<a title="click for an explanation" href="#bd"><span title="go back to the text">Back</span></a>]</p>
    <p>&nbsp;</p>
    <p><sup>e<a name="e" id="e"></a> </sup>Other techniques, such as <a title="go for more information" href="water_methods.html#nmr">NMR</a>, are incapable of showing this. <a href="water_methods.html#tera">Terahertz</a> and <a href="water_methods.html#vib">Infrared Spectroscopy</a> show that the hydration water with retarded reorientational dynamics extends to only about four H<sub>2</sub>O layers (~0.85 nm)   [<a title="go to cited reference" href="ref29.html#r2848">2848</a><a name="r2848" id="r3"></a>]. [<a title="click for an explanation" href="#be"><span title="go back to the text">Back</span></a>] 
      <!-- print end -->
    </p>
    <div class="spacer">
      <p>&nbsp;</p>
</div>
</div>
</div>
</div>
<div style="width:761px; background-color:#006868;padding-top:0px;padding-bottom:3px;border:0px;margin:0px;">
  <div style="width:753px; background-image:url('images/bg2.gif');margin-left:3px;padding-left:2px;border:0px;padding-top:0px;padding-bottom:0px;background-color:#D8F0d8;">
    <p>&nbsp;</p>
    <p align="center"><span class="style3"><a title="Home page" href="index.html">Home</a> | <a title="go to our 'Site index'" href="water_structure_science.html" accesskey="I">Site Index</a> | <a href="protein_denatured.html">Protein folding and denaturation</a> | <a title="go to more on polysaccharide hydration" href="polysaccharide_hydration.html">Polysaccharide 
      hydration</a> | <a title="go to more on nucleic acid hydration" href="nucleic_acid_hydration.html">Nucleic acid hydration</a> | <a title="go to find more on ionic effects" href="hofmeister_series.html">Hofmeister 
      effect</a> | <a href="kosmotropes_chaotropes.html" title="more on kosmotropes and chaotropes">Kosmotropes and chaotropes</a> | <a href="intracellular_water.html">Intracellular water</a> | <a href="http://www.lsbu.ac.uk/" ><abbr title="London South Bank University">LSBU</abbr></a> | <a title="go to top of page" href="#top">Top</a></span></p>
    <p>&nbsp;</p>
    <p align="center"><span class="style3">This page was established in 2001 and  last updated 
        by <a title="email me" href="mailto:martin.chaplin@btinternet.com?subject=from_water_web_site">Martin Chaplin</a> on
        <!-- #BeginDate format:Sw1 -->15 November, 2021<!-- #EndDate -->
</span></p>
    <br />
    <div class="floatleft" style="width:320px;">
      <div align="right"><a rel="license" href="https://creativecommons.org/licenses/by-nc-nd/2.0/uk/"><img alt="Creative Commons License" style="border-width:0" src="https://licensebuttons.net/l/by-nc-nd/2.0/uk/88x31.png" /></a></div>
</div>
   <span class="style4">This work is licensed under a <a rel="license" href="https://creativecommons.org/licenses/by-nc-nd/2.0/uk/">Creative Commons Attribution<br />
    -Noncommercial-No Derivative Works 2.0 UK: England &amp; Wales License</a></span>
    <p>&nbsp;</p>

    <div class="spacer"></div>
</div>
</div>
</body>
</html>
